### PR TITLE
Header layout fixes

### PR DESCRIPTION
**Pull request by jsiems**
Other small layout changes as well.

### CODE REVIEW

**Comment by jsiems** [comment]
Desktop has an account menu that slides out when you move your mouse over the account button, and mobile has one that pops up when you click on the account button. 

Check it out at [ama-jsiems.c9users.io](https://ama-jsiems.c9users.io)

File: public/css/desktop.css [hunk 1]
```diff
@@ -280,61 +276,23 @@ a {
...
+	top: 75px;
 	left: 50%;
 	transform: translateX(-50%);
 	z-index: 2;
-	height: 30px;
 	padding: 10px;
-	padding-top: 60px;
 	display: flex;
 	background-color: #42474B;
     justify-content: center; /* align horizontal */
```
Comment (s1037989) [inline public/css/desktop.css:293]: Be consistent with your spacing.

File: public/css/style.css [hunk 2]
```diff
@@ -29,21 +30,84 @@ body {
...
+	right: 20px;
+	z-index: 3;
+	color: #c5c5c5;
+	background-color: #42474B;
+	border-radius: 15px;
+	transition: right 0.5s ease-in-out, top 0.5s ease-in-out;
+}
+
+.title {
+    width: 95%;
```
Comment (s1037989) [inline public/css/style.css:68]: Be consistent with your spacing, not just within a single block, but for all of your code in all files.  One style to rule them all.

**Comment by s1037989** [comment]
:+1:

**Comment by jsiems** [comment]
It's strange, in my cloud nine environment, all spacing is the same, but in GitHub some things are spaced differently causing spacing discrepancies. What would cause that?

**Comment by s1037989** [comment]
Spaces versus tabs!

**Comment by meyerben** [comment]
:+1: I'll also leave this video here [Tabs versus Spaces](https://youtu.be/SsoOG6ZeyUI)

**Comment by jsiems** [comment]
I have never used spaces though. Could it be caused by the program auto-tabbing when I make a newline? Maybe when it autotabs it uses spaces, compared to when I manually tab it does a tab. Or maybe it converts them to spaces when I copy and paste something.

**Comment by s1037989** [comment]
Yes, absolutely.  Play around with C9's IDE options / settings.  It could be that you use tabs and then C9's autoindent uses spaces...

**Pull request merged**